modules/islandora_altmetrics/README.md

```diff
@@ -0,0 +1,72 @@
+## Introduction
```

Comment:
ack, good catch. Killed the title, didn't I?

Just pushed a commit back to fix that.